jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/quicknewjob/NewJobPresenter.java

```diff
@@ -0,0 +1,182 @@
+            jobCtxMap.put(RequestDataSetConstants.COLUMN_BUSINESSKEY,
+                          jobName); // TODO make legacy keys hard to repeat by accident
+
+            executorServices.call(
+                    (Long requestId) -> {
+                        displayNotification(constants.RequestScheduled(requestId));
+                        requestCreatedEvent.fire(new RequestChangedEvent(requestId));
+                        view.hide();
+                    },
+                    (Message message,
+                     Throwable throwable) -> {
+                        view.showBasicPane();
+                        if (isInvalidCommandTypeError(throwable)) {
+                            view.showInvalidTypeErrorMessage();
+                        } else {
+                            view.showInlineNotification(throwable.getCause().getMessage());
+                        }
+                        return false;
+                    }).scheduleRequest(serverTemplateId,
+                                       jobType,
+                                       dueDate,
+                                       jobCtxMap);
+        } else {
+            view.showBasicPane();
+        }
+
+    }
+
+    protected boolean validateForm(String jobName, String jobType, String jobRetries) {
+        boolean valid = true;
+        view.cleanErrorMessages();
+        if (jobName.isEmpty()) {
+            view.showNameErrorMessage();
+            valid = false;
+        }
+
+        if (jobType.isEmpty()) {
+            view.showEmptyTypeErrorMessage();
+            valid = false;
+        }
+
+        if (jobRetries.isEmpty()) {
+            view.showRetriesErrorMessage();
+            valid = false;
+        }
+
+        return valid;
+    }
+
+    private boolean isInvalidCommandTypeError(Throwable throwable) {
+        String message = "";
+        if (throwable.getMessage() != null) {
+            message = throwable.getMessage();
+        } else if (throwable.getCause() != null) {
+            message = throwable.getCause().getMessage();
+        }
+        return message.contains("Invalid command type");
+    }
+
+    private void displayNotification(String text) {
+        notification.fire(new NotificationEvent(text));
+    }
+
+    @Inject
+    public void setNotification(final Event<NotificationEvent> notification) {
+        this.notification = notification;
+    }
+
+    @Inject
+    public void setRequestChangedEvent(final Event<RequestChangedEvent> requestChangedEvent) {
+        this.requestCreatedEvent = requestChangedEvent;
+    }
+
+    @Inject
+    public void setExecutorService(final Caller<ExecutorService> executorService) {
+        this.executorServices = executorService;
+    }
+
+    public interface NewJobView extends UberElement<NewJobPresenter> {
+
+        void show();
+
+        void hide();
+
+        void showBasicPane();
+
+        void cleanErrorMessages();
+
+        void showNameErrorMessage();
```

Comment:
Maybe to be clearer and consistent with two types of errors for _Type_ field, we could state in the method name what type of error is for others too, i.e. ```showEmptyNameErrorMessage()``` and below ```showEmptyRetriesErrorMessage()```?